package.json

```diff
@@ -144,6 +152,11 @@
         "category": "Metals",
         "title": "Restart server"
       },
+      {
+        "command": "metals.build-restart",
```

Comment:
Not sure about the name so that it's not mixed with `Server restart`. Maybe `Bloop restart`?

Reply:
`bloop-restart` is probably a better name.

Reply:
Changed to "Restart Bloop server"